site/docs/v1/tech/apis/themes.adoc

```diff
@@ -69,6 +70,65 @@ include::docs/v1/tech/apis/_theme-response-body.adoc[]
 include::docs/v1/tech/apis/_theme-response-body-suffix.adoc[]
 {%capture singleResponse %}false{% endcapture %}
 
+== Create a New Theme From an Existing Theme
+
+This API is used to create a new Theme from an existing Theme.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Create a new Theme with a randomly generated Id
+[.endpoint]
+.URI
+--
+[method]#POST# [uri]#/api/theme#
+--
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Create a Theme with the provided unique Id
+[.endpoint]
+.URI
+--
+[method]#POST# [uri]#/api/theme/`\{themeId\}`#
+--
+
+==== Request Parameters
+
+[.api]
+[field]#themeId# [type]#[UUID]# [optional]#Optional# [default]#defaults to secure random UUID#::
+The Id to use for the new Theme. If not specified a secure random UUID will be generated.
+
+==== Request Body
+
+[.api]
+[field]#sourceThemeId# [type]#[UUID]# [required]#Required#::
+The Id of an existing Theme to make a copy of.
++
+If present, the [field]#defaultMessages#, [field]#localizedMessages#, [field]#templates#, and [field]#stylesheet# from the source Theme will be copied to the new Theme. All other fields not listed here may be provided on this request, such as [field]#theme.name#.  The [field]#theme.name# is required.
```

Comment:
I don't think `theme.name` should be mentioned here because it's got another field listed on the request body.

The "if present" doesn't make sense anymore because we are calling this out as a separate API, so `sourceThemeId` will always be present when using this API.

Are there other fields that can be specified? It seems like the only other field is `data`, which isn't worth calling out IMO.

```suggestion
The [field]#defaultMessages#, [field]#localizedMessages#, [field]#templates#, and [field]#stylesheet# from the source Theme will be copied to the new Theme.
```

Reply:
I agree - `data` can be left out.